src/components/shared/PaymentInterval.vue

```diff
@@ -1,48 +1,40 @@
 <template>
```

Comment:
You shouldn't need to change `PaymentInterval` at all, everything that's required for it is in the new `RadioField` component. So in the donation form's `Payment` component you can swap the `PaymentInterval` with a `RadioField`.

A nice way of making sure the new one is working properly is to add the new `RadioField` next to the old one, give it a temporary different name, and hook it up to the model. Changing one field should then change both and the store.